container_pipeline/models/pipeline.py

```diff
@@ -23,7 +23,7 @@ class Project(models.Model):
     service
     """
     name = models.CharField(max_length=200, unique=True, db_index=True)
-
+    target_file_link = models.CharField(max_length=350, default=None, blank=True, null=True)
```

Comment:
Can you please make sure that this line is less than 80 characters long? It's causing the [CI to fail](https://ci.centos.org/job/centos-container-pipeline-service-ci-pr/1970/console).